### DISCUSSION TITLE
Microsoft Edge Add-ons store developer policies are impossible to follow

### DISCUSSION THREAD

**Discussion by loftyshaky** [microsoft/MicrosoftEdge-Extensions#71] [Q&A]
Hello, my update to extension for Edge browser was rejected with the following message:

Content Policies

2.13 Videos
If you submit a promotional Youtube video in the store listing, it should follow all the content guidelines mentioned in this policy. You also need to ensure that advertisements are disabled for the specific videos that you want to be embed. Here are the links to understand how ads are enabled/disabled on Youtube: https://support.google.com/youtube/answer/2531367?hl=en&ref_topic=7072227 & https://support.google.com/youtube/answer/132596?hl=en

I wrote to reportapp@microsoft.com asking what's wrong with my video and they told me that my video starts with ads. They linked to the same Google articles which describe how to disable monetization for a specific video.

The problem: my channel is not monetized, but Youtube still shows ads in my videos. Even more, I don't have the option to disable ads as described in these Google articles. Apparently it's only possible to disable monetization (ads) for monetized channel. See this screenshot: https://i.imgur.com/A9kPw2n.png

When I explained this to reportapp@microsoft.com and asked what should I do they responded with "We do not have any additional information to provide".

My question is, how do I comply with such unfollowable policy? Should I just remove the the video from listing page (I don't want to do this since the video shows how to use the extension)?

And the other question is, why even make such impossible-to-follow policy?

I know you shouldn't post questions related to a specific developer account or extension here but this problem affects every developer with small Youtube channel (there is no way to monetize channel until you meet subscriber/watch hours requirements btw) that posts video on listing page. Make the policy followable.

Extension's id: ifphophaconbhfmkpdlfldelkjpjmlbj

**Comment by sivsouvamMSFT** [comment]
Hi @loftyshaky, thank you for sharing the concern. We will look into it & get back to you with a solution.

**Reply by LucasAschenbach** [reply]
I am facing the exact same issue. Has there been any progress on this?

**Comment by aakarim** [comment]
Hi, we're in exactly the same boat. We don't have monetization enabled and can't disable ads on our Youtube videos.

We would be happy to switch to something else like Vimeo, but we can only use Youtube links on the Edge store.
